client.go

```diff
@@ -269,6 +269,8 @@ func parsePayload(b []byte) (*Conn, error) {
 	if err != nil {
 		return conn, err
 	}
+	conn.Dst = nil
+	conn.Src = nil
```

Comment:
Were these two not enough to solve the problem?

Reply:
It was not. What was necessary was creating a new []byte. I just initialized them to nil just in case.

all I know is that now I don't see them anymore.

at the very least, this change definitely does _not_ introduce bugs.

Reply:
ack. Thanks!